### ISSUE TITLE
Add a str().from_radix(radix) method

### ISSUE THREAD

**Issue by cojmeister** [pola-rs/polars#6533]
### Problem description

I would like to have a function that allows to parse strings from any base (radix) to decimal - and the opposite method as well.
For example:
```rust
let df = df! [
    "A"        => ["110", "010", "101"],
    ]?;

df
  .lazy()
  .withColumn(
      col("binary")
      .str()
      .from_radix(2)
      .alias("decimal")
  )
  .collect()?
```

```
┌──────────────┬─────────┐
│ binary       ┆ decimal │
│ ---          ┆ ---     │
│ str          ┆ i64     │
╞══════════════╪═════════╡
│ "110"        ┆ 6       │
├╌╌╌╌╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌┤
│ "010"        ┆ 2       │
├╌╌╌╌╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌┤
│ "101"        ┆ 5       │
└──────────────┴─────────┘
```

Once this is completed I can add it to the nodejs-polars repo

**Comment by cojmeister** [comment]
I tried starting an implementation - but I can't seem to make it work - any adivce will be recommended.
I added the following function to `trait Utf8NameSpaceImpl` in `polars/polars-ops/src/chunked_array/strings/namespace.rs`:
```rust
...
    #[cfg(feature = "string_encoding")]
    fn from_str_radix(&self, radix: Option<u32>) -> Int64Chunked {
        let ca = self.as_utf8();

        let f = |s: &str| <i64 as Num>::from_str_radix(s, radix.unwrap_or(2)).unwrap();
        ca.apply_cast_numeric(f)
    }
...
```

I'm not sure how to go forward from here; or even if I'm in the right place...

**Comment by cojmeister** [comment]
Attempted solving this in #6570 
Seems to work but waiting for advice.

**Comment by cojmeister** [comment]
Solved in #6570
